### PR TITLE
User related templates

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-494
| Versions      | 3.3+

### CODE REVIEW

File: docs/guide/content_rendering/layout/add_forgot_password.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Add forgot password option
+
+You can add a "forgot password" option to the site front to allow users of a specific SiteAccess, both admin or front, request a password change by customizing the template used in `/user/forgot-password`.
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/content_rendering/layout/add_forgot_password.md:3]: Actually, no. I think the route is available out of the box, but you only customize the template? So maybe something like:
```suggestion
The "forgot password" option allows users of a specific SiteAccess, admin or front, to request a password change.
You can customize the template used in the `/user/forgot-password` route.
```

File: docs/guide/content_rendering/layout/add_login_form.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
+# Add user login form
+
+You can add a login form for your users. The template defines lots of config options for example, information about password expiration or [other user management templates](../../user_management/user_management.md#other-user-management-templates).
```
Comment (DominikaK) [inline docs/guide/content_rendering/layout/add_login_form.md:3]:
```suggestion
You can add a login form for your users. The template defines lots of config options for example, information about password expiration or [other user management templates](../../user_management/user_management.md#other-user-management-templates).
```
The second sentence is unclear to me. Does the template *define* options, or can it use them?

File: docs/guide/content_rendering/layout/add_login_form.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+If you only want to change a template, in the `config/packages/views.yaml` add the following code:
+
+```yaml
+ezpublish:
+    system:
+        my_siteaccess:
+            user:
+                login_template: '@ezdesign/Security/login.html.twig'
+```
+To add a link redirecting to the login form, in the page layout template, provide the following code:
```
Comment (DominikaK) [inline docs/guide/content_rendering/layout/add_login_form.md:16]:
```suggestion

To add a link redirecting to the login form, in the page layout template, provide the following code:
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/guide/content_rendering/layout/add_login_form.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    </p>
+    <p>
+        <a href="{{ path('ezplatform.user.forgot_password') }}" class="btn btn-primary ez-btn ez-btn--login">
+            {{ 'authentication.credentials_expired.reset_password'|trans|desc('Reset password') }}
+        </a>
+    </p>
+{%- endblock -%}
+```
+
+## Customize login form
```
[the remark this answers is not in the document]
Comment (juskora) [reply docs/guide/content_rendering/layout/add_login_form.md:48]: Fixed, copying error.

File: docs/guide/content_rendering/layout/add_login_form.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
+# Add user login form
+
+You can add a login form for your users. 
+Follow the instruction below to create a template with login form. If you want to configure more options for example, password expiration, see [other user management templates](../../user_management/user_management.md#other-user-management-templates).
```
Comment (DominikaK) [inline docs/guide/content_rendering/layout/add_login_form.md:4]:
```suggestion
Follow the instruction below to create a template with login form. If you want to configure more options, for example, password expiration, see [other user management templates](../../user_management/user_management.md#other-user-management-templates).
```

**Review by DominikaK — APPROVED** [review]

**Pull request merged**